source/Octopus.Tentacle/Tentacle.exe.nlog

```diff
@@ -12,7 +12,7 @@
   </extensions>
 
   <variable name="appName" value="Octopus Tentacle" />
-  <variable name="messageLayout" value="${message}${onexception:${newline}${exception:format=ToString}}"/>
+  <variable name="messageLayout" value="${longdate} | ${message}${onexception:${newline}${exception:format=ToString}}"/>
```

Comment:
Do we need to make this change? Lens has a way to show timestamps

![CleanShot 2024-02-20 at 16 55 35@2x](https://github.com/OctopusDeploy/OctopusTentacle/assets/2888279/62176238-84bb-4ad1-af44-885d9204e4f5)

Reply:
Oh I didn't know! Thanks, I'll revert this change then 👍

Reply:
Done 👍